content/home/talks.md

```diff
@@ -28,4 +28,5 @@ Conference call
 September 11, 2019 (10 am - 12 pm)   
 **[Join Zoom](https://oregonstate.zoom.us/j/3571021423    )**
 Meeting ID: 357 102 1423  
-**Agenda:** Coming soon 
+**Agenda:**         
```

Comment:
plz change to `**[Agenda:](http://cvpia.scienceintegrationteam.com/meetings/#agenda )**`